### ISSUE TITLE
Manifest merger failed : uses-sdk:minSdkVersion 15 cannot be smaller than version 16 declared in library 

### ISSUE THREAD

**Issue by technotomahi** [jeduan/cordova-plugin-facebook4#685]
**Cordova build android failed**

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processDebugManifest'.
> Manifest merger failed : uses-sdk:minSdkVersion 15 cannot be smaller than version 16 declared in library [:CordovaLib] <My-platform-path>\android\CordovaLib\build\intermediates\manifests\full\debug\AndroidManifest.xml as the library might be using APIs not available in 15
        Suggestion: use a compatible library with a minSdk of at most 15,
                or increase this project's minSdk version to at least 16,
                or use tools:overrideLibrary="org.apache.cordova" to force usage (may lead to runtime failures)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

**Comment by peterpeterparker** [comment]
Duplicate https://github.com/jeduan/cordova-plugin-facebook4/issues/661

**Comment by erikwitteveen** [comment]
There is a very simple fix for this, when using plugins some of those might be outdated in your platforms folder check if all the plugins are working on version 19 and not for example version 17 or 15. a quick guide to help you is written below 👍 

to find the android version folder go to: platforms => android => AndroidManifest.xml (make sure its 19)

to find your outdated plugins folders go to: platforms => android => cordova-something-something

change the outdated values to 19 and you should be good to go :)

![image](https://user-images.githubusercontent.com/48087035/57022231-ce7b6700-6c2e-11e9-8d65-a06071853f46.png) 
![image](https://user-images.githubusercontent.com/48087035/57022284-f4087080-6c2e-11e9-8695-5f5520fbf779.png)

**Comment by acedesigns** [comment]
In case Someone come across the same issue. Because the solution by @erikwitteveen did not make sense.

In my `config.xml` I added the following 

````
. . .
<preference name="SplashScreen" value="screen" />
<preference name="SplashScreenDelay" value="3000" />
<platform name="android">
     <preference name="android-minSdkVersion" value="19" />
     <preference name="android-targetSdkVersion" value="28" />
     <edit-config file="app/src/main/AndroidManifest.xml" mode="merge" target="/manifest/application" xmlns:android="http://schemas.android.com/apk/res/android">
            <application android:networkSecurityConfig="@xml/network_security_config" />
     </edit-config>
. . .
````
My Environment

```
Ionic:

   Ionic CLI                     : 6.12.0 (/usr/local/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 5.5.1
   @angular-devkit/build-angular : 0.1000.8
   @angular-devkit/schematics    : 10.0.8
   @angular/cli                  : 10.0.8
   @ionic/angular-toolkit        : 2.3.3

Cordova:

   Cordova CLI       : 9.0.0 (cordova-lib@9.0.1)
   Cordova Platforms : android 8.1.0
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.2.0, cordova-plugin-ionic-webview 4.2.1, (and 11 other plugins)

Utility:

   cordova-res (update available: 0.15.3) : 0.15.1
   native-run (update available: 1.3.0)   : 1.0.0

System:

   Android SDK Tools : 26.1.1 (/Users/anele/Library/Android/sdk)
   ios-deploy        : 1.11.1
   NodeJS            : v14.4.0 (/usr/local/Cellar/node/14.4.0/bin/node)
   npm               : 6.14.4
   OS                : macOS Catalina
   Xcode             : Xcode 11.7 Build version 11E801a
```
